### PR TITLE
Add tvb-framework

### PR DESCRIPTION
**Pull request by umarbrowser**

### CODE REVIEW

**Comment by umarbrowser** [comment]
@johanneskoester @ocefpaf @jrbourbeau @CurtLH @CJ-Wright Please can you review and merge  this it took time that i can remember

File: recipes/tvb-framework/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+{% set name = 'tvb-framework' %}
+{% set version = '1.5.5' %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  md5: 3f501c3f43705440f8953bf24aa6b0e5
```
Comment (ocefpaf) [inline recipes/tvb-framework/meta.yaml]: Please remove this we only use `sha256`.

File: recipes/tvb-framework/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  sha256: 81c6b293bff080f4b52ec2c9ece5a3e109c951eb9dd42b1d90aed00589d4828b
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/t/tvb-framework/{{ name }}-{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipes/tvb-framework/meta.yaml]: There is a mix of using jinja var and spelling the name out in this URL. Please choose one.
(See the example folder.)

File: recipes/tvb-framework/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (ocefpaf) [inline recipes/tvb-framework/meta.yaml]: Please use `pip`

https://conda-forge.org/docs/meta.html#use-pip

File: recipes/tvb-framework/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  skip: True  # [py>27]
+
+requirements:
+  build:
+    - python
+    - setuptools
```
Comment (ocefpaf) [inline recipes/tvb-framework/meta.yaml]: `pip` installs `setuptools`, no need to add it here.

File: recipes/tvb-framework/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    - sqlalchemy
+    - tvb-library
+    - tvb-data
+    - psycopg2
+    
+test:
+  commands:
+    - pytest --pyargs tvb.tests.framework.interfaces.web.controllers.burst_controller_test
+
+
```
Comment (ocefpaf) [inline recipes/tvb-framework/meta.yaml]: Please remove this extra line.

File: recipes/tvb-framework/meta.yaml (hunk 1 above)
Comment (umarbrowser) [reply recipes/tvb-framework/meta.yaml]: Ok

File: recipes/tvb-framework/meta.yaml (hunk 2 above)
Comment (umarbrowser) [reply recipes/tvb-framework/meta.yaml]: got it

**Comment by umarbrowser** [comment]
can you check again

File: recipes/tvb-framework/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+    
+  run:
+    - python
+    - beautifulsoup4
+    - cfflib
+    - cherrypy
+    - formencode
+    - genshi
+    - h5py
+    - Networkx
```
Comment (ocefpaf) [inline recipes/tvb-framework/meta.yaml]: In conda all the packages names are lower case. This is not a big deal and you can change it later. (That is a lot of dependencies BTW, can you double check if that is correct?)

**Comment by umarbrowser** [comment]
Thanks

File: recipes/tvb-framework/meta.yaml (hunk 6 above)
Comment (umarbrowser) [reply recipes/tvb-framework/meta.yaml]: Thanks this is typos is networkx not Networkx

**Pull request merged**